### PR TITLE
Test timetest MVP

### PR DESCRIPTION
**Pull request by vurusovs**
Implemented pytest tests to run timetests executables:
1. No comparison with references
2. No dump to DB

### CODE REVIEW

File: tests/time_tests/test_runner/conftest.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+
+@pytest.fixture(scope="session")
+def executable(request):
+    """Fixture function for command-line option."""
+    return request.config.getoption('executable')
+
+
+@pytest.fixture(scope="session")
+def niter(request):
```
Comment (asomsiko) [inline tests/time_tests/test_runner/conftest.py:60]: I can't find where it is used. Why do we need it?

File: tests/time_tests/test_runner/conftest.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    """
+    with open(metafunc.config.getoption('test_conf'), "r") as file:
+        test_cases = yaml.load(file, Loader=yaml.FullLoader)
+    if test_cases:
+        metafunc.parametrize("instance", test_cases)
+
+
+def pytest_make_parametrize_id(config, val, argname):
+    """Pytest hook for user-friendly test name representation"""
+    return " {0}:{1} ".format(argname, val)
```
Comment (asomsiko) [inline tests/time_tests/test_runner/conftest.py:81]: I'm thinking that argname can be eliminated for shorter test name. What do you think?

File: tests/time_tests/test_runner/test_config.yml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+- device:
+    name: "CPU"
+  model:
+    path: "/nfs/inn/proj/vdp/vdp_tests/stress_tests/master_04d6f112132f92cab563ae7655747e0359687dc9/caffe/FP32/alexnet/alexnet.xml" # TODO: add link to `test_data` repo model
```
Comment (asomsiko) [inline tests/time_tests/test_runner/test_config.yml:4]: I thought we had a way to eliminate internal infrastructure paths? Also running those paths on windows is impossible.

File: tests/time_tests/test_runner/test_timetest.py [hunk 4]
```diff
@@ -1 +1,39 @@
-#TODO: fill
\ No newline at end of file
+"""Main entry-point to run timetests tests.
```
Comment (asomsiko) [inline tests/time_tests/test_runner/test_timetest.py:1]: miss copyright

**Review by asomsiko — CHANGES_REQUESTED** [review]

File: tests/time_tests/test_runner/conftest.py (hunk 1 above)
Comment (vurusovs) [reply tests/time_tests/test_runner/conftest.py:60]: It's used here - https://github.com/openvinotoolkit/openvino/blob/7ad3a7feb2a6abf551349c50953dabddc3ad5302/tests/time_tests/test_runner/test_timetest.py#L36

Required for run_timetest

File: tests/time_tests/test_runner/conftest.py [hunk 5]
```diff
@@ -0,0 +1,81 @@
+"""
```
[the remark this answers is not in the document]
Comment (vurusovs) [reply tests/time_tests/test_runner/conftest.py:4]: Added

File: tests/time_tests/test_runner/test_timetest.py (hunk 4 above)
Comment (vurusovs) [reply tests/time_tests/test_runner/test_timetest.py:4]: Added

File: tests/time_tests/scripts/requirements.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+PyYAML==5.1
```
[the remark this answers is not in the document]
Comment (vurusovs) [reply tests/time_tests/scripts/requirements.txt]: Fixed 5.3.1 version

File: tests/time_tests/test_runner/conftest.py [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+
+
+def pytest_generate_tests(metafunc):
+    """Pytest hook for test generation.
+
+    Generate parameterized tests from discovered modules and test config
+    parameters.
+    """
+    with open(metafunc.config.getoption('test_conf'), "r") as file:
+        test_cases = yaml.load(file, Loader=yaml.FullLoader)
```
[the remark this answers is not in the document]
Comment (vurusovs) [reply tests/time_tests/test_runner/conftest.py]: Updated

File: tests/time_tests/test_runner/requirements.txt [hunk 8]
```diff
@@ -0,0 +1,3 @@
+pytest==4.0.1
+attrs==19.1.0   # required for pytest==4.0.1 to resolve compatibility issues
+PyYAML==5.1
```
[the remark this answers is not in the document]
Comment (vurusovs) [reply tests/time_tests/test_runner/requirements.txt]: Fixed 5.3.1 version

File: tests/time_tests/test_runner/conftest.py (hunk 2 above)
Comment (vurusovs) [reply tests/time_tests/test_runner/conftest.py]: Done. Now keys of all internal dicts will be eliminated. Example: device: {name: CPU} -> device_CPU

File: tests/time_tests/test_runner/test_config.yml (hunk 3 above)
Comment (vurusovs) [reply tests/time_tests/test_runner/test_config.yml]: Added `expand_env_vars` and converted path to share with env variable. New version is supported by Win too

**Review by asomsiko — APPROVED** [review]

**Pull request merged**